gem/resources/js/Config.js

```diff
@@ -84,5 +84,7 @@
     config.buildArtifacts.automatorJSON         = tmpDir + "/automator.json";
     config.buildArtifacts.automatorScenarioJSON = tmpDir + "/automatorScenarios.json";
     config.buildArtifacts.intendedTestList      = tmpDir + "/intendedTestList.json";
+    
+    config.automatorModality                    = "init";
```

Comment:
We need to comment this.
> This line is a placeholder so that the automator can communicate its current operation to the entire set of Illuminator javascript extensions.  This is because the logical first choice for implementing this -- automator state -- is not available from Extensions.js